src/server/render/tapestry-render.js

```diff
@@ -74,6 +75,18 @@ export default async (path, query, config, headers) => {
     })
     componentData = normalizeApiResponse(data, route)
   }
+
+  if (route.goudaEndpoint) {
```

Comment:
Hey @romulodl, can we change this to something more generic such as `nonCacheableEndpoint`, and also update the README with the new property